SimPEG/Directives.py

```diff
@@ -253,8 +253,7 @@ def endIter(self):
 class Update_IRLS(InversionDirective):
 
     eps_min = None
```

Comment:
@fourndo : could you add docs to these at some point? just in comment form is fine `eps_min = None # short description here`. I will merge this pr though in the mean time.